### PR TITLE
SO-778 - Subcategory pages menu with icons

### PR DESCRIPTION
**Pull request by adohall**

### CODE REVIEW

**Comment by adohall** [comment]
Commits complete. Lucy's suggested change has been made. Ready for review and merge.

File: app/templates/components/dds/main-nav.hbs [hunk 1]
```diff
@@ -2,15 +2,35 @@
...
   {{#each items as |item|}}
     {{#unless item.hide}}
-      <li class={{if (eq auth.currentRouteName item.route) "active"}}>
-        {{#if item.param}}
-          {{ link-to item.label item.route item.param }}
-        {{else if item.query}}
-          {{# link-to route=item.route query=item.query }}
-            {{item.label}}
-          {{/ link-to }}
+      <li class={{if (or (eq auth.currentRouteName item.route) item.subitems) (concat (if (eq auth.currentRouteName item.route) "active ") (if item.subitems "sub-nav"))}} onclick={{action "toggle"}}>
```
Comment (armansa) [inline app/templates/components/dds/main-nav.hbs:5]: The first "if" condition seems un necessary to me! You could use empty string for else instead.

Comment (adohall) [reply app/templates/components/dds/main-nav.hbs:5]: I tried leaving out that first test, but it produces an empty class attribute. Not a big deal I know. Just a little more clutter in the DOM.
class={{value}} results in no class attribute at all if value is false or null, but class={{concat x y}} seems to produce class="" even if both x and y are false or null. I can't find a better way to remove the class attribute if both items in concat are empty.

File: app/components/dds/main-nav.js [hunk 2]
```diff
@@ -21,15 +21,134 @@ export default Component.extend({
...
-        query: {
-          term: '',
-          pageNum: 1,
-          services: [],
-          identifiers: [],
-          locations: [],
-          company_size: [],
-          profile: []
-        }
+        description: "Filium morte multavit si sine causa? quae fuerit causa, nollem.",
```
Comment (armansa) [inline app/components/dds/main-nav.js:24]: Do we have the actual content for here ready?

Comment (adohall) [reply app/components/dds/main-nav.js:24]: Not yet. I'll ask Lucy.

Comment (adohall) [reply app/components/dds/main-nav.js]: Lucy says - remove the text until it is decided on by new product. I've removed the text.

**Pull request merged**